man/rgbasm.5

```diff
@@ -369,63 +369,93 @@ equals $roman clz ( n )$.
+This gives them increased precision, at the cost of a smaller range, while remaining far cheaper to manipulate than floating-point numbers (which
+.Nm
+does not support).
+.Pp
+The default precision of all fixed-point numbers is 16 bits, meaning the lower 16 bits are used for the fractional part; so they count in 65536ths of 1.0.
+This precision can be changed with the
 .Fl Q
-command-line option.
-You can also specify a precise fixed-point value by appending a
+command-line option, and/or by
+.Ic OPT Q
+.Pq see Sx Changing options while assembling .
+An individual fixed-point literal can specify its own precision, overriding the current default, by appending a
 .Dq q
-to it followed by the number of fractional bits, such as
-.Ql 12.34q8 .
+followed by the number of fractional bits: for example,
+.Ql 1234.5q8
+is equal to $0004d2_80
+.EQ
+delim $$
+.EN
+($= 1234.5 * 2 sup 8$).
 .Pp
 Since fixed-point values are still just integers, you can use them in normal integer expressions.
-Some integer operators like
-.Sq +
-and
-.Sq -
-don't care whether the operands are integers or fixed-point.
 You can easily truncate a fixed-point number into an integer by shifting it right by the number of fractional bits.
 It follows that you can convert an integer to a fixed-point number by shifting it left that same amount.
 .Pp
 Note that the current number of fractional bits can be computed as
 .Ic TZCOUNT Ns Pq 1.0 .
 .Pp
 The following functions are designed to operate with fixed-point numbers:
-.EQ
-delim $$
-.EN
 .Bl -column -offset indent "ATAN2(y, x)"
 .It Sy Name Ta Sy Operation
-.It Fn DIV x y Ta Fixed-point division $( x \[di] y ) \[mu] ( 2 ^ precision )$
-.It Fn MUL x y Ta Fixed-point multiplication $( x \[mu] y ) \[di] ( 2 ^ precision )$
-.It Fn FMOD x y Ta Fixed-point modulo $( x % y ) \[di] ( 2 ^ precision )$
-.It Fn POW x y Ta $x$ to the $y$ power
-.It Fn LOG x y Ta Logarithm of $x$ to the base $y$
+.It Fn DIV x y Ta Fixed-point division
+.It Fn MUL x y Ta Fixed-point multiplication
+.It Fn FMOD x y Ta Fixed-point modulo
+.It Fn POW x y Ta $x sup y$
+.It Fn LOG x y Ta $log sub y ( x )$ (such that
+.Ic LOG Ns ( Ic POW Ns ( x , y ) , y ) == x)
 .It Fn ROUND x Ta Round $x$ to the nearest integer
-.It Fn CEIL x Ta Round $x$ up to an integer
-.It Fn FLOOR x Ta Round $x$ down to an integer
+.It Fn CEIL x Ta Round $x$ up to the nearest integer
+.It Fn FLOOR x Ta Round $x$ down to the nearest integer
 .It Fn SIN x Ta Sine of $x$
 .It Fn COS x Ta Cosine of $x$
 .It Fn TAN x Ta Tangent of $x$
-.It Fn ASIN x Ta Inverse sine of $x$
-.It Fn ACOS x Ta Inverse cosine of $x$
-.It Fn ATAN x Ta Inverse tangent of $x$
+.It Fn ASIN x Ta Inverse sine of $x$ (such that
+.Ic ASIN Ns ( Ic SIN Ns ( x ) ) == x)
+.It Fn ACOS x Ta Inverse cosine of $x$ (such that
+.Ic ACOS Ns ( Ic COS Ns ( x ) ) == x)
+.It Fn ATAN x Ta Inverse tangent of $x$ (such that
+.Ic ATAN Ns ( Ic TAN Ns ( x ) ) == x)
```

Comment:
From the `LOG` discussion:

> I've seen different notation used for at least the trig functions (`sin-1`, `isin` for “inverse”...), so I think it's warranted there.

There's no ambiguity about the explanation "inverse sine", and just calling it `ASIN` instead of `SINM1` or `ISIN` shows which convention we're using.

Reply:
Having grown up in a country where I wasn't even *aware* that the `arc*` notation *existed*, I'm feeling iffy about this.

The equations help disambiguate that we're talking about “the reverse of `func`” and not “`1 / func`”, since the latter is also called an inverse.

Reply:
I recall a similar incident where you thought that 4 was smaller than 4... I do wonder if your (as in, your area's) math terminology is just uncommon.
Arc sine and friends are the standard English names for these functions — if someone doesn't know them, a web search would answer the question instantly.

Reply:
```suggestion
.It Fn ASIN x Ta Inverse sine of $x$
.It Fn ACOS x Ta Inverse cosine of $x$
.It Fn ATAN x Ta Inverse tangent of $x$
```

Reply:
How about "arc sinus"? :P <https://fr.wiktionary.org/wiki/arc_sinus>